### ISSUE TITLE
How to make a list callout with icons - Dumb question?

### ISSUE THREAD

**Issue by marcolli** [mgmeyers/obsidian-list-callouts#16]
I don't know how to make it work with the icons. Do I need to use other plugins? I have tried with "icon shortcodes" and with "icon plugins" and I have not gotten it to work. 
I'm probably missing something very basic so I would appreciate someone's help 🙏
![image](https://user-images.githubusercontent.com/18728748/189489551-73acaee9-ca6b-4312-9b1d-961908b9b51e.png)

**Comment by berot3** [comment]
No, for me, it simply worked. Try disabling all other plugins except this one, see if it works then.

**Comment by marcolli** [comment]
I finally decided to switch to Minimal theme custom checkboxes for this kind of annotations

I can only hope that soon they will be in obsidian's default theme

ty for the answer

**Comment by mgmeyers** [comment]
@marcolli I'm not sure what issue you're facing here, but you just have to select the icon you wish to use in the settings.
<img width="529" alt="Screen Shot 2022-10-14 at 3 20 38 PM" src="https://user-images.githubusercontent.com/2694747/195952268-cad77018-d561-4de7-8ceb-6e8b0472953f.png">

<img width="124" alt="Screen Shot 2022-10-14 at 3 21 22 PM" src="https://user-images.githubusercontent.com/2694747/195952339-28d58c67-7ed3-4257-ab0f-7a731361b126.png">

**Comment by DryIce1** [comment]
> I don't know how to make it work with the icons. I have tried with "icon shortcodes" 

- yes the solution is simple, don't worry!
- in the list callout settings, put the emoji directly into the text box, and don't choose set icon.

![image](https://user-images.githubusercontent.com/59501376/199422225-1c64cdca-3fdc-4a77-8d32-1ea5e88b9c03.png)
